### DISCUSSION TITLE
Not Wroking

### DISCUSSION THREAD

**Discussion by 62040132** [projectdiscovery/uncover#42] [Q&A] [answered]
### uncover version:
Current Version: v0.0.5

### Current Behavior:
When I run echo org:"Uber" | uncover -e shodan
or 
uncover -e shodan -q 'org:"Uber"'

It doesn't show any result. I have checked my shodan key.
It works on shodan.io, just enter org:"Uber" in search.

### Expected Behavior:
Where is the result?

### Steps To Reproduce:
1. Run uncover -e shodan -q 'org:"Uber"'
2. See result

**Comment by ehsandeep** [comment, marked as answer]
@62040132, you can try running the tool with proxy (burp) to see the actual request, and most probably you will see what's wrong.

```
HTTPS_PROXY=http://127.0.0.1:8080 uncover -q 'org:"Uber"'
```
